src/bin/dual-iir.rs

```diff
@@ -33,8 +33,8 @@ use core::sync::atomic::{fence, Ordering};
 
 use mutex_trait::prelude::*;
 
+use fugit::ExtU32;
```

Comment:
```suggestion
pub use fugit::ExtU32;
```

Reply:
Why does this need to be `pub`? This is in a final `bin` application, so there shouldn't be anyone importing it as a library

Reply:
Right. I thought I was looking at `system_timer`.